src/OpenTelemetry.AutoInstrumentation/OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -60,6 +61,9 @@
 
   <ItemGroup Label="Transient dependencies auto-generated by GenerateNetFxTransientDependencies" Condition=" '$(TargetFramework)' == 'net462' AND $(_IsPacking) != true ">
     <!-- These dependencies are also not included in the NuGet package since the conflicts can be left to NuGet version resolution. -->
+    <PackageReference Include="AWSSDK.Core" />
```

Comment:
Are these OpenTelemetry Instrumentation libraries for AWS?

Reply:
@rajkumar-rangaraj I believe these are dependent packages as [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AWS/OpenTelemetry.Instrumentation.AWS.csproj)

Reply:
Vendor library references should not be a part of this project, as this is against the project's principles. These references could be included in the tests, but not in the main project.

Reply:
@rajkumar-rangaraj TBH, I didn't understand fully what's vendor library, I assumed for each service you can add a package to get trace.  But, I fully understand there are 4 libraries we are adding and bringing those dependencies, so I was asking in the SIG (last 2 meetings) and checking with everyone if it's ok.

The other option for us is to enable via Plugin(I have to validate that too), but in that case it would be in our own repo with many AWS attach(exporter, agent etc), which won't benefit community (AFAIK) . Let's discuss what would be ideal in SIG.

Reply:
So the main issue here is that the [OpenTelemetry.Instrumentation.AWS](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWS/1.1.0-beta.2#dependencies-body-tab) NuGet package references other AWS libraries. For our purposes, this is not acceptable because we should not be bringing in other libraries. Compare this to the [OpenTelemetry.Instrumentation.Quartz](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Quartz/1.0.0-beta.2#dependencies-body-tab) package which only references the [OpenTelemetry.Api](https://www.nuget.org/packages/OpenTelemetry.Api/)

I think this prevents us from taking the change, until the AWS instrumentation package can be rewritten in a way that doesn't require referencing the AWS packages.

Reply:
I understand guys and in line with this.... at the same time let me know of any alternative...